bciers/apps/reporting/src/app/components/additionalInformation/additionalReportingData/AdditionalReportingDataForm.tsx

```diff
@@ -43,7 +41,10 @@ export default function AdditionalReportingDataForm({
 }: AdditionalReportingDataProps) {
   const [formData, setFormData] = useState<FormData>(initialFormData);
 
-  const router = useRouter();
+  // 🛸 Set up routing urls
+  const searchParams = useSearchParams();
```

Comment:
Will search Params work here? 
Additional Reporting Data page doesnt have facility-id in params.
Its url is like http://localhost:3000/reporting/reports/3/additional-reporting-data

Reply:
It does from `bciers/apps/reporting/src/app/components/facility/FacilityEmissionAllocationForm.tsx` button `Save & Continue`